src/components/CTA/CTA.tsx

```diff
@@ -1,36 +1,54 @@
-import React, { FunctionComponent } from 'react';
+import React, { FunctionComponent, MouseEvent } from 'react';
 
 import classnames from 'classnames';
 
 import { BlockHeading } from '../../content-blocks/BlockHeading/BlockHeading';
 import { BlockRichText } from '../../content-blocks/BlockRichText/BlockRichText';
 import { DefaultProps, HeadingType } from '../../types';
 import { Button, ButtonProps } from '../Button/Button';
+import { ButtonType } from '../Button/Button.types';
+import { IconName } from '../Icon/types';
 
 import './CTA.scss';
 
 export interface CTAProps extends DefaultProps {
 	heading: string;
 	headingType: HeadingType;
 	content: string;
-	button: Partial<ButtonProps>;
+	buttonLabel: string;
+	buttonOnClick: (event: MouseEvent<HTMLElement>) => void;
+	buttonIcon?: IconName;
+	buttonType?: ButtonType;
 }
 
-// TODO: Replace Button by BlockButtons
 export const CTA: FunctionComponent<CTAProps> = ({
 	className,
 	headingType = 'h3',
 	heading,
 	content,
-	button,
-}) => (
-	<div className={classnames(className, 'c-cta-item')}>
-		<div className="c-cta__content">
-			<div className="c-content">
-				<BlockHeading type={headingType}>{heading}</BlockHeading>
-				<BlockRichText elements={{ content }} />
-				<Button type="secondary" {...button} />
+	buttonLabel,
+	buttonIcon,
+	buttonOnClick,
+	buttonType,
+}) => {
+	const buttonProps: ButtonProps = {
+		label: buttonLabel,
+		title: buttonLabel,
+		ariaLabel: buttonLabel,
+		onClick: buttonOnClick,
+		icon: buttonIcon,
+		type: buttonType,
+	};
+
+	return (
+		<div className={classnames(className, 'c-cta-item')}>
+			<div className="c-cta__content">
+				<div className="c-content">
+					<BlockHeading type={headingType}>{heading}</BlockHeading>
+					<BlockRichText elements={{ content }} />
+					<Button type="secondary" {...buttonProps} />
```

Comment:
uw secondary type gaat altijd overschreven worden, mss best in uw function params als default zetten.